src/Native/configure.cmake

```diff
@@ -234,6 +234,22 @@ check_cxx_source_compiles(
 
 check_cxx_source_compiles(
     "
+    #include <netinet/tcp.h>
+    int main() { int x = TCP_ESTABLISHED; return x; }
+    "
+    HAVE_TCP_H_TCPSTATE_ENUM
+)
+
+check_cxx_source_compiles(
+    "
+    #include <netinet/tcp_fsm.h>
+    int main() { int x = TCPS_ESTABLISHED; return x; }
+    "
+    HAVE_TCP_FSM_H
+)
+
```

Comment:
I changed the tcp_fsm.h detection, but the Linux header declares an enum for the TCP states, which doesn't work with check_symbol_exists. The docs seemed to indicate that check_cxx_source_compiles was the way to go there, so I kept it.